haystack/components/evaluators/answer_exact_match.py

```diff
@@ -0,0 +1,81 @@
+from typing import Any, Dict, List
+
+from haystack import default_from_dict, default_to_dict
+from haystack.core.component import component
+
+
+@component
+class AnswerExactMatchEvaluator:
+    """
+    Evaluator that checks if the predicted answers matches any of the ground truth answers exactly.
+    The result is a number from 0.0 to 1.0, it represents the proportion of questions where any predicted answer
+    matched one of the ground truth answers.
+    Each question can have multiple ground truth answers and multiple predicted answers.
+
+    Usage example:
+    ```python
+    from haystack.components.evaluators import AnswerExactMatchEvaluator
+
+    evaluator = AnswerExactMatchEvaluator()
+    result = evaluator.run(
+        questions=["What is the capital of Germany?", "What is the capital of France?"],
+        ground_truth_answers=[["Berlin"], ["Paris"]],
+        predicted_answers=[["Berlin"], ["Paris"]],
+    )
+    print(result["result"])
+    # 1.0
+    ```
+    """
+
+    def to_dict(self) -> Dict[str, Any]:
```

Comment:
We can leave out the `to_dict` and `from_dict` implementation here as it is just using the default right?
For example, when `component_to_dict` is used it will automatically fall back to `default_to_dict`: https://github.com/deepset-ai/haystack/blob/f69c3e5cd26046b826927a39cad02af93b2ccbbf/haystack/core/serialization.py#L10

Reply:
Right, I always forget that. Will remove them right away.